lib/livebook/config.ex

```diff
@@ -6,6 +6,8 @@ defmodule Livebook.Config do
           | %{mode: :token, secret: String.t()}
           | %{mode: :disabled}
 
+  @type authentication_mode :: :password | :token | :disabled
```

Comment:
Doesn't seem to be used anywhere :)

Reply:
`AuthPlug.store/3` uses it

Reply:
Oh it was already used, but not defined, got it :D